content/guides/monitors.md

```diff
@@ -49,15 +49,47 @@ on the left. This guide will walk through the configuration of the Metric type.
 
 ### Choose what to monitor
 
-1. Select the metric and scope you want to monitor.
+1. Select the alert type.
```

Comment:
In the UI, step 1 is called **Choose the detection method**. Let's use that here instead of **Select the alert type**.